src/main/resources/templates/fragments/header.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+</head>
+<body>
+<div th:fragment="header(activeTab)">
+    <nav class="navbar navbar-expand-lg navbar-dark" style="background-color: rgb(237, 24, 72);">
+        <div class="container">
+            <a class="navbar-brand" href="#">Luštěniny | Admin</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item" th:classappend="${activeTab == 'cypher'}? 'active' : null">
+                        <a class="nav-link" href="/admin/cypher">Cyphers<span class="sr-only">(current)</span></a>
+                    </li>
+                    <li class="nav-item" th:class="${activeTab == 'team'}? 'active' : null">
+                        <a class="nav-link" href="/admin/team">Teams</a>
+                    </li>
+                    <li class="nav-item" th:class="${activeTab == 'finalplace'}? 'active' : null">
+                        <a class="nav-link" href="/admin/finalplace/">Final Place</a>
+                    </li>
```

Comment:
V menu se pouziva anglictina a ve formluarich cestina - chtelo by to sjednotit - asi na to vytvorime issue, co? ;)

Reply:
A ja bych to na cestinu prehodil zrovna, ne? At se to neprotahuje.

Reply:
Urcite zrovna ;)